perun-core/src/main/java/cz/metacentrum/perun/core/provisioning/HierarchicalHashedDataGenerator.java

```diff
@@ -72,19 +75,21 @@ public HashedGenData generateData() {
 		List<Resource> resources =
 				sess.getPerunBl().getFacilitiesManagerBl().getAssignedResources(sess, facility, null, service);
 
-		List<GenDataNode> childNodes = resources.stream()
-				.map(this::getDataForResource)
-				.collect(Collectors.toList());
+		Map<Integer, GenDataNode> childNodes = resources.stream()
+				.collect(toMap(Resource::getId, this::getDataForResource));
 
 		List<String> facilityAttrHashes = dataProvider.getFacilityAttributesHashes();
```

Comment:
``facilityAttrHashes`` is not used anywhere.

Reply:
Variable removed.